Jenkinsfile_CNP

```diff
@@ -69,6 +69,7 @@ withPipeline(type, product, component) {
     loadVaultSecrets(serviceSecrets + otherSecrets)
   }
   onPR {
+    enableMutationTest()
```

Comment:
This seems to be doing the opposite of what the PR says it should be doing. Surely all you need to do is disable it in the nightly file?

Reply:
Yes, I am trying to see if this is going to enable on PR and then I will remove from nightly pipeline :D, sorry it's sort of in the draft mode.